src/ext/audio_ports_config.rs

```diff
@@ -42,6 +45,21 @@ pub struct clap_plugin_audio_ports_config {
         Option<unsafe extern "C" fn(plugin: *const clap_plugin, config_id: clap_id) -> bool>,
 }
 
+#[repr(C)]
+#[derive(Debug, Copy, Clone)]
+pub struct clap_plugin_audio_ports_config_info {
+    pub current_config: Option<unsafe extern "C" fn(plugin: *const clap_plugin) -> clap_id>,
+    pub get: Option<
+        unsafe extern "C" fn(
+            plugin: *const clap_plugin,
+            config_id: clap_id,
+            port_index: u32,
+            is_input: bool,
+            config: *mut clap_plugin_audio_ports,
```

Comment:
This is still wrong, it needs to be `clap_audio_port_info`: https://github.com/free-audio/clap/blob/065d685d4e9657f0344f350eef748be2b4d8e318/include/clap/ext/audio-ports-config.h#L90